WooCommerce/WooCommerceTests/POS/Mocks/MockCartViewModel.swift

```diff
@@ -7,27 +7,14 @@ class MockCartViewModel: CartViewModelProtocol {
     lazy var cartSubmissionPublisher = cartSubmissionSubject.eraseToAnyPublisher()
     let cartSubmissionSubject: PassthroughSubject<[CartItem], Never> = .init()
 
-    lazy var addMoreToCartActionPublisher = addMoreToCartActionSubject.eraseToAnyPublisher()
```

Comment:
This was one of those places that indicated that something is overcomplicated with our current architecture. I'm glad we don't need it anymore.

As I understand, moving away from `observeTotalsOrderActions` is also in the plans? Which eventually will remove the need for `startNewOrder` within `PointOfSaleDashboardViewModel`?